Makefile

```diff
@@ -45,7 +45,7 @@ OPENSHIFT_GOIMPORTS_BIN := openshift-goimports
 OPENSHIFT_GOIMPORTS := $(TOOLS_DIR)/$(OPENSHIFT_GOIMPORTS_BIN)-$(OPENSHIFT_GOIMPORTS_VER)
 export OPENSHIFT_GOIMPORTS # so hack scripts can use it
 
-GOLANGCI_LINT_VER := v1.49.0
+GOLANGCI_LINT_VER := v1.50.1
```

Comment:
Do we also need to set `GOMEMLIMIT` in the Makefile to influence behavior in the `test-e2e*` targets?

Reply:
I'm not sure that we have to, at least in the Makefile. We can probably set `GOMEMLIMIT` in the e2e environment directly if we have ways to test memory consumption?